### PR TITLE
Feature/service status edit

### LINKED ISSUE #168: User unable to edit 'Service' status
Steps: 

1.Visit https://dashboard.digipalvelutehdas.fi   
2.Login  
3.Create Organization  
4.Go to Organization view and select the organization added above     
5.Create Department  
6.Select the department added above  
7.Add Service  
8.Click on service added above  
9.Click on 'Edit'  and update service status 

Result: User not allowed to edit service status      

Operating system: 
- Linux : Ubuntu 14.04.03 LTS (Trusty)

Web Browsers:
- Google Chrome 48.0.2564.103    

Screenshot:      

unable-to-edit-servicestatus

### PR DESCRIPTION
**Pull request by Alapan**
Closes #168 

# Proposed changes
(List any significant changes or additions here)
- I added the meteor-x-editable-bootstrap package for creating an inline select field.

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

**Comment by brylie** [comment]
Is this still a work in progress, or should we review it now?

**Comment by Alapan** [comment]
@brylie This can be reviewed now.

File: services/client/single/single.js [hunk 1]
```diff
@@ -45,6 +45,8 @@ Template.singleService.events({
 
     // Get reference to template instance
     const instance = Template.instance();
+    // Reference to array holding allowed status values
+    let options = [];
```
Comment (brylie) [inline services/client/single/single.js]: Rename this variable to something more explicit. E.g. `statusOptions`

**Comment by Alapan** [comment]
@brylie The change was made.

**Pull request merged**